### ISSUE TITLE
whether it is more harder to train if neck & ears been added?

### ISSUE THREAD

**Issue by wangxinjun56** [AaronJackson/vrn#137]

**Comment by AaronJackson** [comment]
Probably depends on the quality of the dataset. I'm not aware of any large face scan datasets which will work well on in the wild images, which also have properly aligned necks and ears.

**Comment by wangxinjun56** [comment]
i make samples like this,  is it will be more harder to train?  :
![QQ图片20191015144224](https://user-images.githubusercontent.com/15605685/66806424-14688500-ef5a-11e9-9589-e9d4d480a6bf.jpg)
![QQ图片20191015144235](https://user-images.githubusercontent.com/15605685/66806456-2518fb00-ef5a-11e9-8cf5-cdd4c70cd6a2.jpg)

**Comment by AaronJackson** [comment]
It looks ok, but I can only see one sample. The most important thing is for the data to be consistent throughout the whole dataset.

**Comment by wangxinjun56** [comment]
consistent  is impossible, as the pose and size etc, are not the same;
but they are look like the same change rules，so，is it can be train good ?

**Comment by AaronJackson** [comment]
Well, you'd probably get better performance by fixing the size. It's common to normalise the size of the face in some way regardless of what you are doing. Pose will of course need to vary if you want it to work on different poses. 

If you do what I did in the paper, it should train fine.

**Comment by wangxinjun56** [comment]
thanks ,i will normalise the size， just now i tryed  one hourglass ; it looks  like it ' s not to be convergenced, have you tryed one hourglass ?

**Comment by wangxinjun56** [comment]
i used 100 samples, two hourglass, loss=sigmoid_cross_entropy_with_logits,   (256*256*200)
the result like this :
![QQ截图20191017083626](https://user-images.githubusercontent.com/15605685/66968815-6ecc2780-f0b9-11e9-80b6-ad084a4018c4.png)
![QQ截图20191017083644](https://user-images.githubusercontent.com/15605685/66968821-725fae80-f0b9-11e9-96e1-4bf222dece2b.png)
![QQ截图20191017083656](https://user-images.githubusercontent.com/15605685/66968825-75f33580-f0b9-11e9-8c4e-e04c0b6ad996.png)
![QQ截图20191017083715](https://user-images.githubusercontent.com/15605685/66968828-78558f80-f0b9-11e9-9e09-24b5d9ef9cec.png)

**Comment by wangxinjun56** [comment]
![QQ截图20191017084818](https://user-images.githubusercontent.com/15605685/66969128-e8b0e080-f0ba-11e9-8ef9-0e1209b3f656.png)


the sum loss , is the sum of all losses 256*256*200;   so even if the sum loss to be trained to very samll, but can't promise the every one loss is the same small .

like this:
s = (x+y+z)/3
we make s is small ; but may be x is so big ,and y is so small ...not promis x and y are the same small

if s = 1; then x =1, y=1,z=1;
also could be , x = 2,y=0.5,z=0.5;
so the x is not samll loss
so the vol can't be train good .

i 'm just a litte think about it ;may be i'm wrong

**Comment by AaronJackson** [comment]
100 training samples? No chance

**Comment by wangxinjun56** [comment]
so ,how many samples at least?

**Comment by huyanfei-cqupt** [comment]
> 
> 
> so ,how many samples at least?

@wangyinjun56 Hi, After reading your comments, I have some questions that I would like to share with you; I would like to ask you some questions, and I hope you can help me answer them. Is it convenient to leave your email address?  Looking forward to your reply. thaks

**Comment by gernhard1337** [comment]
Hello. 
@wangxinjun56 could you be so kind and share your training code? i am kinda new to this topic and i am not sure how to train it myself. I got a very detailed database and wanted too see if a trained model with high resolution data can improve output.

**Comment by AaronJackson** [comment]
Just closing off some old issues.
